tools/WebViewSample/src/README.md

```diff
@@ -0,0 +1,14 @@
+# README
+
+## Overview
+This project provides a test app that opens a web page in the [Microsoft.Web.WebView2](https://www.nuget.org/packages/Microsoft.Web.WebView2) control. It is used as a test bed for running Accessibility Insights for Windows on a web application. Thsi is generaly _not_ recommneded, since it is difficult to know how to address any issues that are raised. The recommended option is to scan the HTML using a web-based engine such as Accessibility Insights for Web.
```

Comment:
typo: `... Thsi is generaly _not_ recommneded...` -> `... This is generally _not_ recommended...`

Reply:
Fixed :)